newrelic/structures_newrelic_alert_muting_rule.go

```diff
@@ -386,3 +387,36 @@ func flattenWeeklyRepeatDays(daysOfWeek []alerts.DayOfWeek) []string {
 
 	return out
 }
+
+func handleMutingRuleCreateUpdateError(err error) diag.Diagnostics {
+	var diags diag.Diagnostics
+
+	if graphQLError, ok := err.(*alerts.GraphQLErrorResponse); ok {
```

Comment:
```suggestion
        var graphQLError *alerts.GraphQLErrorResponse
	if errors.As(err, &graphQLError)
```
Using the following code would be a better approach:

Reply:
Thank you, was a very valuable suggestion 🙂 changed this in the latest commit.